EKSD_LATEST_RELEASES

```diff
@@ -12,7 +12,6 @@ releases:
   number: 9
   kubeVersion: v1.21.5
 - branch: 1-22
-  number: 6
+  number: 1
   kubeVersion: v1.22.6
-  dev: true
 latest: 1-21
```

Comment:
Should we bump this now? This basically effects the builds who pull in kubectl into their images and don't care about a specific version.

Reply:
I think we can start testing with our prod release just to make sure right?

Reply:
ok, well update this before we ship?